datasets/indic_glue/indic_glue.py

```diff
@@ -0,0 +1,889 @@
+_WSTP_LANGS = ["as", "bn", "gu", "hi", "kn", "ml", "mr", "or", "pa", "ta", "te"]
+_iNLTKH_LANGS = ["gu", "ml", "mr", "ta", "te"]
+_BBCA_LANGS = ["hi"]
+_CVIT_MKB_CLSR = ["en-bn", "en-gu", "en-hi", "en-ml", "en-mr", "en-or", "en-ta", "en-te", "en-ur"]
+_IITP_MR_LANGS = ["hi"]
+_IITP_PR_LANGS = ["hi"]
+_ACTSA_LANGS = ["te"]
+_MD_LANGS = ["hi"]
+_WIKI_NER_LANGS = ["as", "bn", "gu", "hi", "kn", "ml", "mr", "or", "pa", "ta", "te"]
+
+_NAMES = []
+
+for lang in _WNLI_LANGS:
+    _NAMES.append(f"wnli.{lang}")
+
+for lang in _COPA_LANGS:
+    _NAMES.append(f"copa.{lang}")
+
+for lang in _SNA_LANGS:
+    _NAMES.append(f"sna.{lang}")
+
+for lang in _CSQA_LANGS:
+    _NAMES.append(f"csqa.{lang}")
+
+for lang in _WSTP_LANGS:
+    _NAMES.append(f"wstp.{lang}")
+
+for lang in _iNLTKH_LANGS:
+    _NAMES.append(f"inltkh.{lang}")
+
+for lang in _BBCA_LANGS:
+    _NAMES.append(f"bbca.{lang}")
+
+for lang in _CVIT_MKB_CLSR:
+    _NAMES.append(f"cvit-mkb-clsr.{lang}")
+
+for lang in _IITP_MR_LANGS:
+    _NAMES.append(f"iitp-mr.{lang}")
+
+for lang in _IITP_PR_LANGS:
+    _NAMES.append(f"iitp-pr.{lang}")
+
+for lang in _ACTSA_LANGS:
+    _NAMES.append(f"actsa-sc.{lang}")
+
+for lang in _MD_LANGS:
+    _NAMES.append(f"md.{lang}")
+
+for lang in _WIKI_NER_LANGS:
+    _NAMES.append(f"wiki-ner.{lang}")
+
+
+class IndicGlueConfig(datasets.BuilderConfig):
+    """BuilderConfig for IndicGLUE."""
+
+    def __init__(self, data_url, citation, url, text_features, **kwargs):
+        """
+        Args:
+
+          data_url: `string`, url to download the zip file from.
+          citation: `string`, citation for the data set.
+          url: `string`, url for information about the data set.
+          text_features: `dict[string, string]`, map from the name of the feature
+        dict for each text field to the name of the column in the csv/json file
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(IndicGlueConfig, self).__init__(version=datasets.Version("1.0.0", ""), **kwargs)
+        self.data_url = data_url
+        self.citation = citation
+        self.url = url
+        self.text_features = text_features
+
+
+class IndicGlue(datasets.GeneratorBasedBuilder):
+
+    BUILDER_CONFIGS = [
+        IndicGlueConfig(
+            name=name,
+            description=_DESCRIPTIONS[name.split(".")[0]],
+            text_features=_TEXT_FEATURES[name.split(".")[0]],
+            data_url=_DATA_URLS[name.split(".")[0]],
+            citation=_CITATIONS[name.split(".")[0]],
+            url=_URLS[name.split(".")[0]],
+        )
+        for name in _NAMES
+    ]
+
+    def _info(self):
+        features = {text_feature: datasets.Value("string") for text_feature in six.iterkeys(self.config.text_features)}
+
+        if (
+            self.config.name.startswith("wnli")
+            or self.config.name.startswith("copa")
+            or self.config.name.startswith("actsa")
+        ):
+            features["label"] = datasets.Value("int32")
```

Comment:
Could you change the labels fields to be of type `datasets.Sequence(datasets.features.ClassLabel(...))` ?
You can find an example in [glue sst2](https://github.com/huggingface/datasets/blob/master/datasets/glue/glue.py#L174) for example in which for each task the labels classes titles are specified to define the `label` feature type (see [here](https://github.com/huggingface/datasets/blob/master/datasets/glue/glue.py#L457-L458))

Reply:
Updated as requested for **actsa** and **wnli**

the labels for **copa** as index so that requires the format to be **int32**, so that remained unchanged

Reply:
Yes right.
Also we don't have a proper convention for multi-choice QA yet, so expect it to change at one point.